packages/system/velero/Makefile

```
@@ -9,3 +10,18 @@ update:
 	helm repo add tanzu https://vmware-tanzu.github.io/helm-charts
 	helm repo update tanzu
 	helm pull tanzu/velero --untar --untardir charts
+	tag=$$(yq .image.tag charts/velero/values.yaml) && \
+	sed -i "/ARG VERSION/ s|=.*|=$${tag}|g" images/velero/Dockerfile
+
+image:
```

checkmake (0.2.2)

[warning] Target body for "image" exceeds allowed length of 5 (11).

(maxbodylength)